sublime_linter.py

```diff
@@ -312,11 +313,22 @@ def get_linters_for_view(view):
     bid = view.buffer_id()
     current_linters = persist.view_linters.get(bid, [])
 
-    wanted_linters = []
-    for linter_class in persist.linter_classes.values():
-        settings = linter_module.get_linter_settings(linter_class, view)
-        if linter_class.can_lint_view(view, settings):
-            wanted_linters.append(linter_class(view, settings))
+    filename = view.file_name()
+    # Unassign all linters from orphaned views
+    if filename and not os.path.exists(filename):
+        logger.info(
+            "Skipping buffer {}; '{}' is unreachable".format(bid, filename))
+        flash_once(
+            none_for_none(lambda: view.window().id()),
+            "'{}' has become unreachable.".format(filename)
```

Comment:
For status_messages skip the quotes around the path and the period around the end.

Reply:
Okidoki, you're in detail mode. 😁

Reply:
Gotta feel like I contribute something 😀